my_game.py

```diff
@@ -17,14 +17,17 @@
 SCREEN_HEIGHT = 600
 
 # Variables controlling the player
-PLAYER_LIVES = 3
 PLAYER_SPEED_X = 5
 PLAYER_START_X = SCREEN_WIDTH / 2
 PLAYER_START_Y = 50
+PLAYER_LIVES = 3
+PLAYER_SPEED = 3
```

Comment:
_PLAYER_SPEED_ is not used. Please remove it. I think your editor also warns you about this.